x-pack/platform/plugins/shared/stack_connectors/public/connector_types/lib/servicenow/helpers.test.ts

```diff
@@ -51,6 +53,17 @@ describe('helpers', () => {
     });
   });
 
+  describe('isCORSError', () => {
+    test('should return true if the error is CORSError', async () => {
```

Comment:
Do we need async here and line 62?

Reply:
No, that was a bad copy-paste 🙂. Thanks, I will update it.

Reply:
Done [`e382903` (#213658)](https://github.com/elastic/kibana/pull/213658/commits/e382903e9149a71523de7afe7192ed513337a316).